meta-mender-overo-sdcard/recipes-bsp/u-boot/u-boot-overo.inc

```diff
@@ -0,0 +1,16 @@
+FILESEXTRAPATHS_prepend_overo := "${THISDIR}/patches:"
+
+# Overo board is using 2016.11 u-boot, need to apply a patch that matches our version
+SRC_URI_remove_overo += " file://0003-Integration-of-Mender-boot-code-into-U-Boot.patch"
+SRC_URI_append_overo += " file://0003-U-Boot-pre-2017.03-integration-of-Mender-boot-code-i.patch"
+
+# This will likely not work at the moment.  The default assumes the UBoot environment
+# is in NAND.  For proper SDCard based Mender updates, that should probably be moved
+# to MMC so we can easily update it.
+# UBI is currently tested and working; this is being kept for archival purposes.
```

Comment:
This comment confused me :). The commit message says that it is SD card integration? And you seem to have moved env to MMC based on the u-boot patch?

Reply:
+1, also confused! Maybe it can be removed, since you are including the UBI based integration as well?

Reply:
It's both.   I started the SDCard integration but didn't finish it but figured it may be worth archiving here.

Reply:
I'll cleanup the commit logs a bit to make it more evident.